### PR TITLE
[Wakatime Card] - Add languages filter and topN for wakatime card

### LINKED ISSUE #696: [API][Wakatime Card] - Add languages filter and topN for wakatime card
**Describe the solution you'd like**

I found wakatime card show too many languages, such as `html`, `toml`, etc.

Look this

image

So I want to add languages filter and topN for wakatime card,Like this: 

```
/api/wakatime?username=francis&hide=HTML,TOML,Bash,JAVA,RUST,Markdown&top=5
```

### PR DESCRIPTION
**Pull request by francis-du**
Fix #696 

URL 

```
https://github-readme-stats-git.francisdu.vercel.app/api/wakatime?username=francis&hide=other,html,scss,css,yaml,xml,toml&top=5
```

Preview

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "dotenv": "^8.2.0",
     "emoji-name-map": "^1.2.8",
     "github-username-regex": "^1.0.0",
+    "node-fetch": "^2.6.1",
```
Comment (nathanchu) [inline package.json:30]: Why was node-fetch added? I can't seem to find where it was used and we already have axios for data fetching

File: api/wakatime.js [hunk 2]
```diff
@@ -34,7 +36,9 @@ module.exports = async (req, res) => {
   }
 
   try {
-    const last7Days = await fetchLast7Days({ username });
+    const last7Days = await fetchLast7Days(username, top, hide);
+
+    console.log(last7Days)
```
Comment (nathanchu) [inline api/wakatime.js:41]: `console.log` here is unneeded.

File: package.json (hunk 1 above)
Comment (nathanchu) [reply package.json:30]:
```suggestion
```

File: api/wakatime.js (hunk 2 above)
Comment (nathanchu) [inline api/wakatime.js:41]:
```suggestion
```

Comment (nathanchu) [reply api/wakatime.js:41]: `console.log` here is unneeded.

File: package.json (hunk 1 above)
Comment (francis-du) [reply package.json:30]: I will remove it.

File: api/wakatime.js (hunk 2 above)
Comment (francis-du) [reply api/wakatime.js:41]: Yes

**Comment by francis-du** [comment]
Done

**Comment by nathanchu** [comment]
Looks good to me, could you just improve the tests so they pass.

**Comment by francis-du** [comment]
Sure

**Comment by francis-du** [comment]
Done.

**Comment by francis-du** [comment]
Hi! @anuraghazra Counld you please review this PR, I think it's a good feature.

**Comment by WilsonCazarre** [comment]
Any status update on this PR? Does it need more improvement and/or help in some way?

**Comment by francis-du** [comment]
I don’t know, I think he shouldn’t have time to deal with PRs.But this can be used on my own deployed node.

`https://github-readme-stats-git.francisdu.vercel.app`

**Comment by WilsonCazarre** [comment]
Thanks, I'll give it a try.

**Pull request closed without merging**